packages/react-integration/demo-app-ts/src/components/demos/NotificationDrawerDemo/NotificationDrawerGroupsDemo.tsx

```diff
@@ -42,68 +41,41 @@ export class GroupsNotificationDrawerDemo extends React.Component<
   constructor(props: NotificationDrawerProps) {
     super(props);
     this.state = {
-      isOpen: false,
-      isActive: '',
-      firstGroupIsOpen: false,
-      secondGroupIsOpen: true,
-      thirdGroupIsOpen: false
+      isOpenMap: null,
+      firstGroupExpanded: false,
+      secondGroupExpanded: true,
+      thirdGroupExpanded: false
     };
   }
 
-  onToggle = (isOpen: boolean) => {
+  onToggle = (id: string, isOpen: boolean) => {
     this.setState({
-      isOpen
+      isOpenMap: { [id]: isOpen }
     });
   };
-
   onSelect = (event: any) => {
     this.setState({
-      isOpen: !this.state.isOpen,
-      isActive: ''
-    });
-    this.onFocus(event.target.id);
-  };
-
-  onClick = (event: any) => {
-    this.setState({
-      isActive: event.target.id
+      isOpenMap: null
     });
-    this.onFocus(event.target.id);
   };
-
-  onFocus = (id: string) => {
-    if (id) {
-      const element = document.getElementById(id);
-      element.focus();
-    }
-  };
-
   toggleFirstDrawer = (event: any, value: boolean) => {
```

Comment:
Can these handlers be typed as `React.SyntheticEvent<HTMLDivElement>`?

Reply:
Fixed as above.